### ISSUE TITLE
Color wrong on PAL 576i videos

### ISSUE THREAD

**Issue by AliciaTywysoges** [iina/iina#3664]
Hi, there seems to be an issue with colors on 576i content, tried many 576i videos but IINA screenshots show normal colors, have tried turning on and off hardware decoder, force GPU with no avail, even tried on my old macbook
Left being in IINA and right being the screenshot taken from IINA https://imgur.com/pFtyDFq

video info sample
Video
ID                                       : 1
ID in the original source medium         : 224 (0xE0)
Format                                   : MPEG Video
Format version                           : Version 2
Format profile                           : Main@Main
Format settings                          : CustomMatrix / BVOP
Format settings, BVOP                    : Yes
Format settings, Matrix                  : Custom
Format settings, GOP                     : M=3, N=7
Format settings, picture structure       : Frame
Codec ID                                 : V_MPEG2
Codec ID/Info                            : MPEG 1 or 2 Video
Duration                                 : 21 min 29 s
Bit rate mode                            : Variable
Bit rate                                 : 5 626 kb/s
Maximum bit rate                         : 8 300 kb/s
Width                                    : 720 pixels
Height                                   : 576 pixels
Display aspect ratio                     : 4:3
Frame rate mode                          : Constant
Frame rate                               : 25.000 FPS
Standard                                 : PAL
Color space                              : YUV
Chroma subsampling                       : 4:2:0
Bit depth                                : 8 bits
Scan type                                : Interlaced
Scan order                               : Bottom Field First
Compression mode                         : Lossy
Bits/(Pixel*Frame)                       : 0.543
Time code of first frame                 : 00:00:00:00
Time code source                         : Group of pictures header
GOP, Open/Closed                         : Closed
Stream size                              : 865 MiB (97%)
Language                                 : English
Default                                  : No
Forced                                   : No
Color primaries                          : BT.601 PAL
Transfer characteristics                 : BT.470 System B/G
Matrix coefficients                      : BT.601
Original source medium                   : DVD-Video

**System and IINA version:**

- macOS  12.2.1
- IINA  Latest

**Expected behavior:**

**Actual behavior:**

<details>
<summary><b>Crash report:</b></summary>

```
```
</details>

<br>

<details>
<summary><b>mpv log:</b></summary>

```
```
</details>

<br>

**Steps to reproduce:**

- [ ] MPV does not have this problem.

**How often does this happen?**

**Comment by AliciaTywysoges** [comment]
I can supply the videos if thats of any use.

**Comment by low-batt** [comment]
Hello from another IINA user! I looked at the pictures posted on imgur. That looks really broken. 

The issue template asks this:
```text
- [ ] MPV does not have this problem.
```

Did you try playing the offending file with the [mpv media player](https://mpv.io/), and if so did you see the same darkness?

From the mpv homepage:
> mpv is a free (as in freedom) media player for the command line.

And:
> While mpv strives for minimalism and provides no real GUI, it has a small controller on top of the video for basic control. 

In addition to the command line player the mpv project also provides libmpv which can be embedded in other applications. IINA is one of the [GUI frontends](https://github.com/mpv-player/mpv/wiki/Applications-using-mpv#gui-frontends) for mpv. So for playback problems IINA wants to know that mpv player does not have the problem because if it fails in the mpv player then it is likely the problem is not in IINA code and must be fixed "upstream" by the mpv project.

You could also try playing the offending video with [IINA+](https://github.com/iina-plus/iina). That is an **unofficial** fork of IINA. Think of it as a "developers playground". A major emphasis for that fork is testing potential changes to support HDR, so display issues are being discussed in that fork. Any problems with IINA+ should be discussed in the issue section of that fork.

**Comment by zknpr** [comment]
@AliciaTywysoges  I have the same problem, how did you solve it?

**Comment by AliciaTywysoges** [comment]
@zknpr Nope, never resolved the problem, nor were my other issues with Iina ever resolved even after supplying information multiple times in the past so just gave up with support.

**Comment by zknpr** [comment]
sad to hear
